app/_locales/en/messages.json

```diff
@@ -1,14 +1,14 @@
 {
   "appName": {
-    "message": "Behave!",
+    "message": "Behave! — Offensive Comments Blocker",
```

Comment:
Thanks, @iislucas! These are very good. I'll split test them during coming 2 weeks and see which one converts better.